src/base/bittorrent/private/filterparserthread.cpp

```diff
@@ -124,71 +119,108 @@ int FilterParserThread::parseDATFilterFile()
         return ruleCount;
     }
 
-    const QByteArray data = file.readAll();
+    static const int bufferSize = 2 * 1024 * 1024; // 2 MiB
```

Comment:
with `static` or without, the variable is deemed to be optimized out by the compiler

http://stackoverflow.com/a/27466684  (case 1)
https://en.wikipedia.org/wiki/Constant_folding